lib/diagnostic.js

```diff
@@ -6,33 +6,57 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const getNormalizedWords = function (text) {
+  let words = text.split(' ');
```

Comment:
This should be split using the regular expression ```/\s+/``` as its argument rather than a single space.  ```/\s+/``` matches all whitespace (spaces, tabs, line breaks, etc.).